tests/components/axis/test_switch.py

```diff
@@ -31,17 +33,17 @@
 async def test_platform_manually_configured(hass):
     """Test that nothing happens when platform is manually configured."""
     assert await async_setup_component(
-        hass, switch.DOMAIN, {"switch": {"platform": axis.DOMAIN}}
+        hass, SWITCH_DOMAIN, {SWITCH_DOMAIN: {"platform": AXIS_DOMAIN}}
     )
 
-    assert axis.DOMAIN not in hass.data
+    assert AXIS_DOMAIN not in hass.data
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)